_docs/install-and-upgrade.md

```diff
@@ -6,14 +6,27 @@ redirect_from:
   - /docs/getting_started/install.html
 ---
 
+## Installation Methods
+
+* [Debian/Ubuntu-based distros](#debian-and-ubuntu-based-distributions)
+* [Arch Linux](#arch-linux)
+* [NPM Releases](#from-npm-releases)
+* [Docker](#docker)
+* [Running from source](#running-from-source)
+* [Unofficial Install Methods](#unofficial-install-methods)
+* [Services packaging The Lounge](#services-packaging-the-lounge)
```

Comment:
Let me figure out generated tables of contents (not just how to do it, that's the easy part, but if it actually makes sense and how to make them good) before we ship. So I suggest removing this for now, and we'll re-add if we end up not having generated TOCs.

Reply:
Sounds good. The page was getting really long and unwieldy.